src/components/navbar/navbar.component.tsx

```diff
@@ -84,6 +124,30 @@ const Navbar: React.FC<NavbarProps> = ({
                 <Logo />
               </HeaderLink>
               <HeaderGlobalBar>
+                <HeaderGlobalAction className={styles.search} id="searchBar">
+                  <Search
+                    placeHolderText="Search for Patient name or Id"
+                    size="lg"
+                    onBlur={handleSearcbBarClose}
+                  />
+                </HeaderGlobalAction>
+                <HeaderGlobalAction
```

Comment:
Where is the patient search located? Right now its in esm-home; so I'd suggest making this part of an extension living in esm-home.

Reply:
@FlorianRappl  you mean ,the SearchBar should be migrated from `esm-primary-navigation` to `esm-home`

and if that's the case then how do i create the extension ?

Reply:
Please have a look at the extension workshop we did last week. The video link is in Slack.

(TLDR: Extensions are just exported components from a microfrontend)

Reply:
Alright thanks .Let me try to look into it.